charts/cx-operator/templates/configMap.yaml

```diff
@@ -19,4 +19,16 @@ data:
           }
         }
       }
-    }
\ No newline at end of file
+    }
+
+    resources {
+      rulegroups = [
+        {{- range .Values.config.resources.rulegroups }}
+        {
+          namespace = {{ .namespace }}
+          buffer = {{ if .buffer }}{{ .buffer }}{{ else }}{{ $.Values.config.resources.defaultBuffer }}{{ end }}
+        },
```

Comment:
it works with the trailing comma?

Reply:
I think so, but let me double check :)

Reply:
https://github.com/lightbend/config#features-of-hocon

```
Features of HOCON
...
* Allow trailing commas after last element in objects and arrays
```

Reply:
@vigoo cool